content/doc/pipeline/tour/hello-world.adoc

```diff
@@ -29,6 +29,7 @@ Handbook.
 To get started quickly with Pipeline:
 
 . Install the link:https://plugins.jenkins.io/docker-workflow/[Docker Pipeline] plugin through the *Manage Jenkins > Manage Plugins* page.
+- Add the `docker` label to the node you would like to the containers on (can use the controller/built in node)
```

Comment:
@mbaitelman I encountered this yesterday while I was experimenting with #5533.  However, when I tried to duplicate the problem after installing the Docker Pipeline plugin and restarting the controller, I didn't see the issue again.

Can you share any more details about the situation where you found that you needed to add the `docker` label to the agent or the controller?